### PR TITLE
Fix occluder edges for grid rotation

### PR DESCRIPTION
**Pull request by metalgearsloth**
(is eye.Scale the right thing to use for the view matrix?)

### CODE REVIEW

File: Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs [hunk 1]
```diff
@@ -908,10 +909,10 @@ private void UpdateOcclusionGeometry(MapId map, Box2 expandedBounds, Vector2 eye
...
 
                         // Calculate delta positions from camera.
-                        var (dTlX, dTlY) = (tlX, tlY) - eyePosition;
-                        var (dTrX, dTrY) = (trX, trY) - eyePosition;
-                        var (dBlX, dBlY) = (blX, blY) - eyePosition;
-                        var (dBrX, dBrY) = (brX, brY) - eyePosition;
+                        var (dTlX, dTlY) = eyeTransform.Transform(tl);
+                        var (dTrX, dTrY) = eyeTransform.Transform(tr);
+                        var (dBlX, dBlY) = eyeTransform.Transform(bl);
+                        var (dBrX, dBrY) = eyeTransform.Transform(br);
```
Comment (metalgearsloth) [inline Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs:915]: The actual important bit

Comment (metalgearsloth) [inline Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs:915]: The important bit

File: Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs [hunk 2]
```diff
@@ -335,8 +335,9 @@ private void DrawLightsAndFov(Viewport viewport, Box2Rotated worldBounds, Box2 w
             }
 
             var (lights, count, expandedBounds) = GetLightsToRender(mapId, worldBounds, worldAABB);
+            eye.GetViewMatrix(out var eyeTransform, eye.Scale);
```
Comment (PaulRitter) [inline Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs:338]: without having looked at the code: why tf is this using an out var? is this a tryget? if so use it like that

**Review by PaulRitter — APPROVED** [review]
lgtm besides the one nitpick

File: Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs (hunk 2 above)
Comment (metalgearsloth) [reply Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs:338]: It's void for reasons.

Comment (PaulRitter) [reply Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs:338]: weird af

**Pull request merged**